packages/design-system/image/image.tsx

```diff
@@ -10,12 +12,12 @@ export type ImgProps = ExpoImageProps & {
 
 const StyledExpoImage = styled(Image);
 
-type ImageProps = Omit<ExpoImageProps, "resizeMode"> & {
+type ImageProps = Omit<ImgProps, "resizeMode"> & {
   tw?: string;
   alt?: string;
   blurhash?: string;
-  resizeMode?: any;
-} & ImgProps;
+  resizeMode?: ResizeMode;
```

Comment:
Don't you think we should be consistent and change it across the app? Maybe too much effort though

Reply:
due to we are using `expo-av` now and it has a `resizeMode` prop, but i'm also not sure should we continue to use `resizeMode` or use new `contentFit` prop. 
did you prefer to use `contentFit`? happy to change it.

Reply:
Let's keep it for a future task. Right now we have over 102 `resizeMode` across 22 files. We can revise this in a Spring cleaning

Reply:
>Spring cleaning 

🤣 🤣 🤣